src/item/StringToItemParser.php

```diff
@@ -1354,6 +1357,7 @@ private static function registerItems(self $result) : void{
 		$result->register("long_weakness_potion", fn() => Items::POTION()->setType(PotionType::LONG_WEAKNESS()));
 		$result->register("long_weakness_splash_potion", fn() => Items::SPLASH_POTION()->setType(PotionType::LONG_WEAKNESS()));
 		$result->register("magma_cream", fn() => Items::MAGMA_CREAM());
+		$result->register('medicine', fn() => Items::MEDICINE());
```

Comment:
what's the default for this?

Reply:
what type of medicine is given by default? I'm not even sure this alias should be here.